sonar-plugin-api/src/main/java/org/sonar/api/config/PropertyDefinition.java

```diff
@@ -299,6 +299,10 @@ public boolean multiValues() {
     return multiValues;
   }
 
+  /**
+   * @deprecated since 6.1, as it was not used and too complex to maintain.
+   */
+  @Deprecated
   public String propertySetKey() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)